### ISSUE TITLE
[Bug] scrollbar not properly shown when first start / opening file

### ISSUE THREAD

**Issue by makedir** [JasonStein/Notepads#1075]
Showing after first start:

![image](https://user-images.githubusercontent.com/6721083/160053584-1895661d-45d9-492c-b7ad-d67354524b73.png)

This is super annoying. Clicking on it does nothing, you have to hover over it 2 times / click 2 times to get the proper scroll bar:

![image](https://user-images.githubusercontent.com/6721083/160053627-a768d8a0-cddc-4757-876d-329491c4c234.png)

Change this to ALWAYS show the proper scroll bar.

**Comment by JasonStein** [comment]
This is a known issue for the compact overlay feature provided by the OS. Not really a notepads issue.

**Comment by makedir** [comment]
Then implement a workaround. Just let the scrollbar show always, never in the minimalist design, just open the app with normal scrollbars.

**Comment by JasonStein** [comment]
Oh sorry, I replied to the wrong thread. For your issue, the scroll bar is provided by the OS and I don't think I have control over it. I think I can force it to always show but I don't think it is a right thing since:

1. It will cover some of the text.
2. You can change this behavior in system settings:
   
![image](https://user-images.githubusercontent.com/1166162/162550573-1ae21511-e491-44b7-b03a-13121f6c5ff5.png)

**Comment by JasonStein** [comment]
For Win11:

![image](https://user-images.githubusercontent.com/1166162/162550622-8ccacda1-eab0-4f70-b637-84a6f7636816.png)

**Comment by makedir** [comment]
Thanks that actually seems to work as a workaround.

**Comment by makedir** [comment]
Another bug I jut found:

it shows the wrong mouse cursor for the scrollbar. When mouse over it shows the text select cursor not the normal mouse cursor.

**Comment by JasonStein** [comment]
> Another bug I just found:
> 
> it shows the wrong mouse cursor for the scrollbar. When mouse over it shows the text select cursor not the normal mouse cursor.
> 
> ![image](https://user-images.githubusercontent.com/6721083/162551780-8a97acd9-8d33-4138-9eaa-4cf270a084b3.png)

Yes this is one of the bugs I found which I cannot fix since it comes with the default rich edit control UWP provides.
